modules_core/com.smf.securewebservices/src-db/database/model/tables/SMFSWS_CONFIG.xml

```diff
@@ -37,10 +37,18 @@
         <default/>
         <onCreateDefault/>
       </column>
-      <column name="PRIVATEKEY" primaryKey="false" required="true" type="VARCHAR" size="255" autoIncrement="false">
+      <column name="PRIVATEKEY" primaryKey="false" required="true" type="CLOB" size="4000" autoIncrement="false">
```

Comment:
It's change isn't compatible with legacy versions or older versions installed in client environmnents.

Reply:
Why it will be incompatible? If you alter a VARCHAR column to set it as TEXT the data is kept.
Which type of incompatibility you refer?

Reply:
If you are using Postgresql this column will be TEXT but in Oracle database this will be CLOB and this has some operations to use it, may be you need check this in Oracle with data previously created and new data.

Reply:
Oh! Good call! I'll check it

Reply:
To be sure, I've changed it to varchar (2000) due to private keys have 1676 characters, so I set it to 2000 just in case (?